blockchain/api.go

```diff
@@ -875,3 +905,66 @@ func (api *BasicEventsAPI) processLog(log types.Log, eventTS uint64, out chan *E
 		}
 	}
 }
+
+type BasicSimpleGatekeeperApi struct {
+	client             *ethclient.Client
+	gatekeeperContract *marketAPI.Gatekeeper
+	gasPrice           int64
+	logParsePeriod     time.Duration
+}
+
+func NewBasicSimpleGatekeeperApi(client *ethclient.Client, address common.Address, gasPrice int64, logParsePeriod time.Duration) (SimpleGatekeeperAPI, error) {
```

Comment:
I cannot get clearly - it is basic or it is simple? 
Also, `API` is an abbreviation and must be uppercased.

Reply:
Its Basic API for simple gatekeeper 🤦‍♂️